src/Service/ConfigurationService.php

```diff
@@ -345,4 +346,25 @@ public function getAdyenGivingCharityWebsite(string $salesChannelId)
             $salesChannelId
         );
     }
+
+    /**
+     * @param string $salesChannelId
+     * @return array|mixed|null
+     */
+    public function getDomainUrlId(string $salesChannelId): ?string
```

Comment:
According to the previous comment, the name of the method may be changed as `getDefaultDomainId` to make it consistent.